knpu/ansistrano/secret-config.md

```diff
@@ -67,6 +67,16 @@ basically, a hosted MySQL or PostgreSQL database - so that I don't need to manag
 it on my own. In that case, the database host would be something specific to
 my RDS instance. But, it's the same idea.
 
+***TIP
+If you decided to use Amazon's RDS - basically, you need to perform next steps:
+
+1. Create an RDS instance in Amazon Console by hand;
+2. Put its credentials in the Ansible Vault.
+
+Automate RDS instance creation is overkill - databases are not something that should be
+created/recreated like servers because they hold real user data
```

Comment:
Hmm, maybe leave this out. It’s kinda half true I think. Certainly they’re not destroyable like servers. But in a robust setup, you would automate the creating of your *entire* platform, including database servers.

We could say something like:

> You can automate the creation of the RDS instance, though it’s not *as* important because these instances are not destroyed and recreated in the same was as EC2 instances.

Reply:
`was` -> `way`